src/scss/components/_breadbrumbs.scss

```diff
@@ -0,0 +1,52 @@
+//BREADCRUMBS
+
+// Wrapper element.
+.breadcrumbs {
```

Comment:
**I have absolutely no issue with it**, but (I think) I remember the practice team talking about avoiding using the `&__` nested pattern in PostCSS, I assume they'd say the same for SCSS. Again, I have no problem with how this is done here, just thought I'd drop a comment in case there's something I'm missing. 🥂 I could 100% be wrong about the whole thing.